content/docs/examples/_global/nav.md

```diff
@@ -0,0 +1,20 @@
++++
```

Comment:
Same here. Why the additional navigation?

Reply:
It's because of my lack of knowledge about Syna's structure :smile:. Mahdi once told me about this but it seems like I forgot to fix it.